com.github.vladimiry.ElectronMail.json

```diff
@@ -0,0 +1,88 @@
+{
+  "app-id": "com.github.vladimiry.ElectronMail",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "19.08",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "branch": "beta",
```

Comment:
Do you want it to be merged on the beta channel or this is just a leftover?

Reply:
I'd prefer only the beta branch to be created for now since v4.6.0 app release doesn't include the https://github.com/vladimiry/ElectronMail/commit/4fdfa270b630640cf069ad9ae59d5d610ebe9b8a commit. Then I create the master branch when v4.6.1 release is rolled out.